.github/workflows/main.yml

```diff
@@ -125,6 +125,17 @@ jobs:
         run: cd ${{env.dist-directory}} && npm publish --access public
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN_DASCHBOT}}
-      - uses: lakto/gren-action@v1.1.0
+      - if: "github.event.release.prerelease"
+        name: Update release notes
+        uses: lakto/gren-action@v2.0.0
```

Comment:
I do not understand why we use this line and line 137 in dsp-ui

Reply:
You mean the gren-action package? It's a github actions package to use gren, which creates the release notes in github. It was already there before, but now in an updated version to make the difference between the prerelease and the real release. In the old version, it always overwrote the pre-release flag on Github, which is of course bad.
I forked lakto/gren-action from a action script on [Github's marketplace](https://github.com/marketplace?type=actions) to use it with the gren options

Reply:
We use `lakto/gren-action` in all main repos of the DaSCH (dsp-api, dsp-js-lib, dsp-ui-lib and dsp-app) by the way